docs/custom-images-contract.rst

```diff
@@ -95,10 +95,39 @@ Job arguments, in JSON format.
 .. code-block:: javascript
 
     {"key": "1111112/2/2", "project": 1111112, "version": "version1",
-    "spider": "spider-name", "spider_type": "auto", "tags": [],
-    "priority": 2, "scheduled_by": "user", "started_by": "admin",
+    "spider": "spider-name", "spider_type": "auto", "tags": ["tagA", "tagB"],
+    "priority": 2, "scheduled_by": "user", "started_by": "john",
     "pending_time": 1460374516193, "running_time": 1460374557448, ... }
 
+
+Some useful fields
+__________________
+
+============ ======================================================== =================================
+Field        Description                                              Example
+============ ======================================================== =================================
+key          Job key in format ``PROJECT_ID/SPIDER_ID/JOB_ID``        ``"1111112/2/2"``
+project      Integer project ID                                       ``1111112``
+spider       String spider name                                       ``"spider-name"``
+spider_args  Spider args dictionary                                   ``{"arg1":"val1"}``
```

Comment:
In recent discussion we have been talking about using `job_cmd` for passing arguments to custom images even if it is not a script.  spider_args still need to be documented I think.